app/views/application/_repos.html.slim

```diff
@@ -20,4 +21,13 @@ javascript:
     if (hash) {
       $('.nav-tabs a[href='+hash+']').tab('show');
     }
+
+    $(".types-filter-button").click(function() {
```

Comment:
Should this js live somewhere else? Thoughts on using data-roles, id's, or `js-` prefixed classnames for better separation of concerns?